src/uniprotkb/components/entry/NamesAndTaxonomySection.tsx

```diff
@@ -57,6 +68,58 @@ const NamesAndTaxonomySection = ({ data, primaryAccession }: Props) => {
           <GeneNamesView geneNamesData={data.geneNamesData} />
         </>
       )}
+      {data.geneLocations?.length && (
+        <>
+          <h3 data-article-id="encoded_on">Encoded on</h3>
+          <div>
+            {data.geneLocations.map((og) => {
+              let content = og.geneEncodingType;
+              if (og.value) {
+                content += ` ${og.value}`;
+              }
+              let evidence;
+              if (og.evidences) {
+                evidence = <UniProtKBEvidenceTag evidences={og.evidences} />;
+              } else if (references) {
+                // Mapping the evidences to the corresponding encoding type when the evidences are not parsed in the OG object itself (eg. P09130, P04737)
+                const associatedReferences = references.filter((r) =>
+                  r.referenceComments?.some(
+                    (comment) =>
+                      comment.value.toLowerCase() === og.value.toLowerCase() &&
+                      og.geneEncodingType.toLowerCase() ===
+                        comment.type.toLowerCase()
```

Comment:
Minor suggestion to pull out the `og.value.toLowerCase()` and `og.geneEncodingType.toLowerCase()` of the `some` body to avoid repeating `toLowerCase`.